builder/js/ncf.js

```diff
@@ -763,7 +852,13 @@ $scope.groupMethodsByCategory = function () {
   // Resets a Technique to its original state
   $scope.resetTechnique = function() {
     $scope.editForm.$setPristine();
-    $scope.selectedTechnique=angular.copy($scope.originalTechnique);
+    if($scope.tempOriginalTechnique===undefined){
```

Comment:
Still don't understand why you need this variable, would be fixed by having originalTechnique set to tempOriginalTechnique long ago !!